### PR TITLE
Chapter 7: Sign up

### PR DESCRIPTION
**Pull request by ducbm-2610**
## Evidence (Screenshot or Video)
image

BÙI MINH ĐỨC

### CODE REVIEW

**Comment by ducbm-2610** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
2 commit gái ơi

**Comment by ducbm-2610** [comment]
ready

File: config/locales/vi.yml [hunk 1]
```diff
@@ -4,4 +4,6 @@ vi:
     warnings:
       max_length_50: "Quá 50 ký tự"
       min_length_1: "ít nhất 1 ký tự"
-  
+    new:
+      create_my_account: "Tạo tài khoản"
```
Comment (ghost) [inline config/locales/vi.yml:8]: Thiếu endfile em

**Review by ghost — COMMENTED** [review]
fix xong merged nhé

**Pull request merged**